src/tool/cpp/strings/base_error.h

```diff
@@ -177,8 +175,6 @@ WINRT_EXPORT namespace winrt
             WINRT_VERIFY_(impl::error_ok, WINRT_GetRestrictedErrorInfo(m_info.put_void()));
         }
 
-        impl::bstr_handle m_debug_reference;
-        uint32_t const m_debug_magic{ 0xAABBCCDD };
         hresult m_code{ impl::error_fail };
```

Comment:
removed because of BSTR alone?  too bad

Reply:
Restored this. It provides little value as it is rather obscurely supported in Visual Studio, but I'll leave it there until we have a better debugging story.